example/satosa/disco.html

```diff
@@ -148,7 +148,7 @@ <h3>IT Wallet</h3>
                     <div class="row">
                       <div class="col-lg-12 text-center button-container column-buttons">
                           <!-- IDEM IDP BUTTON SMALL "ENTRA CON WALLET ID" * begin * -->
-                          <a href="#" class="btn btn-primary btn-lg btn-me w-100" eidas-idp-button="#eidas-idp-button-xlarge-post" aria-haspopup="false" aria-expanded="false"  style="font-size: 21px;">
+                          <a href="/qr-code-page.html" class="btn btn-primary btn-lg btn-me w-100" eidas-idp-button="#eidas-idp-button-xlarge-post" aria-haspopup="false" aria-expanded="false"  style="font-size: 21px;">
```

Comment:
Non riesco a capire perché le modifiche non vengano testate prima del commit

questo era https://github.com/italia/Satosa-Saml2Spid/blob/eudi/example/static/disco.html#L186

e fino a prova contraria deve rimanere in quella maniera altrimenti la chiamata di ritorno al proxy non funziona.
Su questo problema siete rimasti bloccati per giorni e vi ho sbloccato io con i miei commit, perché adesso stiamo regredendo?

@Pyscho79 ^

Reply:
resolved with https://github.com/italia/eudi-wallet-it-python/pull/75/commits/920db32c73c124860a51ff216134970e34519ae4